source/Optimization.rst

```diff
@@ -0,0 +1,37 @@
+Optimisation
+===============
+
+
+PerTickCache
+----------------
+This function is used to retrieve values from the cache if some value is already computed. If the value is not present in the cache, it computes the value using a user-defined function and stores it in the cache.
```

Comment:
You have forgotten to mention the most important thing! The per-tick cache is only valid for **one tick**!

Reply:
You have mentioned it briefly later, but you should really stress this here as well.